Remotion/Web/UnitTests/Core/WebStringTest.cs

```diff
@@ -0,0 +1,210 @@
+﻿using System;
+using System.IO;
+using System.Threading;
+using System.Web.UI;
+using NUnit.Framework;
+
+namespace Remotion.Web.UnitTests.Core
+{
+  [TestFixture]
+  public class WebStringTest
+  {
+    [Test]
+    public void CreateFromRawAttribute ()
+    {
+      var stringValue = "this & that";
+      var webString = WebString.CreateFromRawAttribute (stringValue);
+
+      Assert.That (webString.Type, Is.EqualTo (WebStringType.Attribute));
+      Assert.That (webString.GetRawValue(), Is.EqualTo (stringValue));
+    }
+
+    [Test]
+    public void CreateFromRawAttribute_WithNullValue_ThrowsArgumentNullException ()
+    {
+      Assert.That (() => WebString.CreateFromRawAttribute (null), Throws.ArgumentNullException);
```

Comment:
Nice attention to detail! Just for reference purposes: typically, we don't bother with testing arg-null and arg-empty cases.